newIDE/app/src/EventsSheet/EventsTree/Renderers/LinkEvent/index.js

```diff
@@ -32,7 +38,6 @@ const styles = {
   title: {
     fontSize: '1.2em',
     whiteSpace: 'nowrap',
-    overflow: 'hidden',
```

Comment:
It was cutting the underlining and the end of the text (because it's in italic), but I bet it's important for something.
Do we need to keep it?

Reply:
I think this is necessary to have the text show the ellipsis if it's larger than the container ("MyVeryLongNaaaaaameeeee...").
I recommend adding just a `paddingRight: 4` to see if it looks better. We could also tweak the lineHeight to set it to '1.3em', give it a try. In any case, let's try to keep this overflow: hidden.

Reply:
Actually, the last character is not cut. I tried with a `d` at the end. The `c` only happens to have a strange shape in italic.